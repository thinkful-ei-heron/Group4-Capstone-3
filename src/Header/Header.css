.header-component {
	background-image: url('../assets/img/hops5.jpg');
	background-size: cover;
}
.header-component h2 {
	color: rgb(0, 0, 0);
	font-family: 'Merriweather', serif;
	font-size: 2em;
	margin-top: 0;
	padding: 1em;
}
.header-component nav a {
	/* padding: 1em; */
	color: white;
	margin: 1em;
	font-size: 1.5em;
	text-decoration: none;
	font-family: 'Merriweather', serif;
}
.header-component nav {
	background-color: rgb(8, 8, 8);
	height: 2.3em;
	text-align: left;
}
.header-component nav a:hover {
	color: rgb(8, 184, 8);
}

.header-component nav button {
	float: right;
	background-color: rgba(8, 8, 8, 0.041);
	color: white;
	font-family: 'Merriweather', serif;
	font-size: 1.5em;
	border: none;
	height: 1.3em;
	margin-right: 1em;
	cursor: pointer;
}
.header-component nav button:hover {
	color: rgb(8, 184, 8);
}
.header-component nav button:hover {
	color: rgb(8, 184, 8);
}
