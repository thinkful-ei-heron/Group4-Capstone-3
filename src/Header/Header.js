import React from "react";
import './Header.css'
import NavBar from '../NavBar/NavBar'

class Header extends React.Component {
    render() {
        return (
            <>
                <section className='header'>
                    <h2>{this.props.header}</h2>
                    <NavBar location={this.props.location}/>
                </section>
            </>
        )
    }
}

export default Header;