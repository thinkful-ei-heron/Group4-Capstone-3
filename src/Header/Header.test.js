  
import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json'

import { BrowserRouter } from 'react-router-dom';
import Header from './Header'

Enzyme.configure({ adapter: new Adapter() });

describe(`Header component`, () => {
it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(
    <BrowserRouter>
      <Header location = {{pathname: ''}}/>
    </BrowserRouter>,
    div)
  ReactDOM.unmountComponentAtNode(div)
})
it('Displays Header when rendered', () => {
  const wrapper = shallow(<Header />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
})