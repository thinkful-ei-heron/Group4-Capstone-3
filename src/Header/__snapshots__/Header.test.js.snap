// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Displays Header when rendered 1`] = `
<Fragment>
  <section
    className="header-component"
  >
    <h2 />
    <NavBar />
  </section>
</Fragment>
`;
